### PR TITLE
Add august meetup post.

### PR DESCRIPTION
**Pull request by ramiroluz**

### CODE REVIEW

**Comment by ellisonleao** [comment]
LGTM

**Comment by ellisonleao** [comment]
@ramiroluz you can add yourself on the authors if you want, right here https://github.com/GruPyPR/grupypr.github.io/blob/src/pelicanconf.py#L63

You will get an authors page with the posts you've written

File: pelicanconf.py [hunk 1]
```diff
@@ -88,6 +88,14 @@
             ('twitter-square', 'https://twitter.com/ellisonleao'),
         )
     }
```
Comment (ellisonleao) [inline pelicanconf.py]: missing `,` here

**Comment by ramiroluz** [comment]
Fixed already.

**Comment by eduardostalinho** [comment]
Who can merge this?

**Comment by ellisonleao** [comment]
@eduardostalinho don't you have the permissions?

**Comment by eduardostalinho** [comment]
Nay!

Em sáb, 11 de nov de 2017 13:35, Ellison Leão <notifications@github.com>
escreveu:

**Comment by ellisonleao** [comment]
@eduardostalinho done. Now you can merge it.

**Comment by eduardostalinho** [comment]
2017-11-13-111553_907x111_scrot

**Comment by ellisonleao** [comment]
You must accept the inviation (which is currently pending right now) . I will merge this and you can continue merging the others.

**Pull request merged**